controlplane/hack/deploy/kustomization.yaml

```diff
@@ -39,5 +40,13 @@ kind: Kustomization
 secretGenerator:
 - name: controlplane-cert
   files:
-  - kube-serviceaccount.key
-  - client-ca.crt
+  - cert/kube-serviceaccount.key
+  - cert/client-ca.crt
+  - cert/client-ca.key
+  - cert/serving-kube-apiserver.crt
+  - cert/serving-kube-apiserver.key
+  - cert/client-kube-aggregator.crt
```

Comment:
we do not need cert/client-kube-aggregator.crt and key, right?

Reply:
we need it in [kubeconfig](https://github.com/open-cluster-management-io/OCM/blob/standalone-controlplane/controlplane/hack/lib/util.sh#L302) which is needed while creating comfigmap  cluster-info, maybe should write kubeconfig with crt and key data rather than file first, then remove the files from secret?

Reply:
We may need to create a separated secret to store kubeconfig. The kubeconfig is consumed by addons

Reply:
As we discussed offline, `kube-aggregator.kubeconfig` will contain ca and token. we create another kubeconfig secret to be used by addons.